kitsune/sumo/static/sumo/js/protocol-details-init.js

```diff
@@ -36,11 +36,10 @@ export default function detailsInit() {
     window.Mzp.Details.init('.details-heading');
     swapMobileSubnavText();
   }
-  _mqWide.addListener(function(mq) {
-    if (sidebarList && mq.matches) {
+  _mqWide.addEventListener("change", function(ev) {
+    if (sidebarList && ev.matches) {
```

Comment:
This change was made simply to avoid using the deprecated `addListener` method.

Reply:
I think the build error on this could be a compatibility issue. Maybe some error handling and/or fallback might get you through circle?

Reply:
I reverted the change.